Application/EdFi.Ods.Api/Caching/PersonUniqueIdToUsiCache.cs

```diff
@@ -108,30 +110,26 @@ public class PersonUniqueIdToUsiCache : IPersonUniqueIdToUsiCache
         /// <returns>The UniqueId value assigned to the person if found; otherwise <b>null</b>.</returns>
         public string GetUniqueId(string personType, int usi)
         {
-            if (usi == default(int))
+            if (usi == default)
             {
-                return default(string);
+                return default;
             }
 
-            if ((_suppressStudentCache && personType == PersonEntitySpecification.Student)
-                || (_suppressStaffCache && personType == PersonEntitySpecification.Staff)
-                || (_suppressParentCache && personType == PersonEntitySpecification.Parent))
+            if (_cacheSuppression.TryGetValue(personType, out bool isSuppressed) && isSuppressed)
```

Comment:
The `ApiSettings` configuration in _appsettings.json_ has been changed for flexibility around person types. So rather than explicit "suppressAbcCache" values, they are now defined by person type in a JSON object, which maps into a .NET Dictionary, keyed by person type.